sway-lsp/src/config.rs

```diff
@@ -109,6 +111,22 @@ impl Default for InlayHintsConfig {
     }
 }
 
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct OnEnterConfig {
+    pub continue_doc_comments: Option<bool>,
+    pub continue_comments: Option<bool>,
+}
+
+impl Default for OnEnterConfig {
```

Comment:
We probably want to update these from VScode? Just a reminder to not forget to put up a PR for the VScode plugin like I did last time :)

Reply:
Good call! I found and fixed a bug in doing so 🐛 

https://github.com/FuelLabs/sway-vscode-plugin/pull/138